src/components/color-picker/color-picker.tsx

```diff
@@ -0,0 +1,106 @@
+import { FormComponent } from '../form/form.types';
+
+/**
+ * This component enables you to select a swatch from out color palette, simply
+ * by clicking on it. You can then copy the css variable name of the chosen color
+ * and use it where desired.
+ *
+ * The color picker can also show you a preview of any valid color name or color value.
+ *
+ * @exampleComponent limel-example-color-picker
+ */
+
+@Component({
+    tag: 'limel-color-picker',
+    shadow: true,
+    styleUrl: 'color-picker.scss',
+})
+export class ColorPicker implements FormComponent {
+    /**
+     * Name or code of the chosen color
+     */
+    @Prop({ reflect: true })
+    public value: string;
+
+    /**
+     * The label of the input field.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    /**
+     * The helper text of the input field.
+     */
+    @Prop({ reflect: true })
+    public helperText: string;
+
+    /**
+     * The title text of the input field.
+     */
+    @Prop({ reflect: true })
+    public tooltipLabel: string;
+
+    /**
+     * Emits chosen value to the parent component
+     */
+    @Event()
+    public change: EventEmitter<string>;
+
+    @State()
+    private isOpen = false;
+
+    public render() {
+        const background = this.value ? { '--background': this.value } : {};
+
+        return [
+            <limel-tooltip
+                label={this.tooltipLabel}
+                elementId="tooltip-button"
+            />,
+            <div class="color-picker">
+                <limel-popover open={this.isOpen} onClose={this.onPopoverClose}>
+                    <div
+                        class="picker-trigger"
+                        slot="trigger"
+                        style={background}
+                        role="button"
+                        tabindex="0"
+                        onClick={this.openPopover}
+                        id="tooltip-button"
+                    />
+                    <limel-color-picker-palette
+                        value={this.value}
+                        label={this.label}
+                        helperText={this.helperText}
+                        onChange={this.onChosenColor}
+                    />
+                </limel-popover>
+
+                <limel-input-field
+                    label={this.label}
+                    helperText={this.helperText}
+                    value={this.value}
+                    onChange={this.handleChange}
+                />
+            </div>,
+        ];
+    }
+    private openPopover = (event: MouseEvent) => {
+        event.stopPropagation();
+        this.isOpen = true;
+    };
+
+    private onPopoverClose = (event: CustomEvent) => {
+        event.stopPropagation();
+        this.isOpen = false;
+    };
+
+    private onChosenColor = (event: any) => {
+        this.value = event.detail;
+    };
+
+    private handleChange = (event: CustomEvent<string>) => {
+        this.value = event.detail;
```

Comment:
Value is a prop so we should not update it ourselves, we should get the value from the consumer.
We need to emit a `change` event here instead and let the consumer know that the value has changed